clang/lib/Driver/ToolChains/Darwin.cpp

```diff
@@ -2031,11 +2053,20 @@ struct DarwinPlatform {
   SourceKind Kind;
   DarwinPlatformKind Platform;
   DarwinEnvironmentKind Environment = DarwinEnvironmentKind::NativeEnvironment;
-  VersionTuple NativeTargetVersion;
-  std::string OSVersion;
-  bool HasOSVersion = true, InferSimulatorFromArch = true;
-  Arg *Argument;
+  // When compiling for a zippered target, this means both target &
+  // target variant is set on the command line, ZipperedOSVersion holds the
+  // OSVersion tied to the main target value.
+  VersionTuple ZipperedOSVersion;
+  // We allow multiple ways to set or default the OS
+  // version used for compilation. When set, UnderlyingOSVersion represents
+  // the intended version to match the platform information computed from
+  // arguments.
```

Comment:
Nit: seems like this can be reflowed.
```suggestion
  // We allow multiple ways to set or default the OS version used for
  // compilation. When set, UnderlyingOSVersion represents the intended version
  // to match the platform information computed from arguments.
```

Reply:
The only change is formatting, right? `clang-format` may be the reason for this, but if not, I'll fix it upstream & cherry-pick.